### PR TITLE
fix(core): breadcrumb keyboard arrow support

### LINKED ISSUE #8565: arrow keyboard support for breadcrumbs
#### Is this a bug, enhancement, or feature request?

enhancement

#### Briefly describe your proposal.

Breadcrumbs should be navigable via keyboard arrows

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes #8565 

Adds arrow support for breadcrumbs. Using this instead of cdk's FocusKeyManager because FocusKeyManager does not cooperate with tab keypresses, nor the functionality with the collapsed breadcrumb menu.

### CODE REVIEW

**Comment by g-cheishvili** [comment]
You could update the active element index on tab navigation for instance and not create arrow navigation

File: libs/core/src/lib/breadcrumb/breadcrumb.component.html [hunk 1]
```diff
@@ -61,8 +61,8 @@
                 fd-link
                 tabindex="0"
                 class="fd-breadcrumb__collapsed"
-                (keydown.enter)="_keyDownHandle($event)"
-                (keydown.space)="_keyDownHandle($event)"
+                (keydown.enter)="_overflowKeyDownHandle($event)"
```
Comment (N1XUS) [inline libs/core/src/lib/breadcrumb/breadcrumb.component.html:64]: Overflow layout supports keyboard navigation via `enableKeyboardNavigation` input property and adding `fdOverflowLayoutFocusableItem` directive to the item that can be focused.

**Review by N1XUS — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/breadcrumb/breadcrumb.component.html (hunk 1 above)
Comment (mikerodonnell89) [reply libs/core/src/lib/breadcrumb/breadcrumb.component.html:64]: This line you've highlighted here is for the overflow menu button. The actual breadcrumb links themselves are coming from the app developer through `ng-content`.  Correct me if I am wrong but I think they would need to provide this directive as well through the `fd-link`s, and import the overflow module, so I think what I have in this PR is simpler for the developer

Comment (N1XUS) [reply libs/core/src/lib/breadcrumb/breadcrumb.component.html:64]: You can provide focusableItem as fd-link. Moreover, you can extend focusableItem directive to search for the focusableItem element inside directives content, which would be much better on my opinion

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
When reviewing this PR keep in mind the following: https://github.com/SAP/fundamental-styles/issues/3821

**Comment by N1XUS** [comment]
@mikerodonnell89 Quick question: If we use key arrows navigation, should we keep tab navigation over items, or just to focus on the first item, and then use arrows only?

**Pull request closed without merging**